### DISCUSSION TITLE
Spatial Seq TCR with 3' chemistry from Hudson Protocol with MiXCR 4

### DISCUSSION THREAD

**Discussion by enricvercher** [milaboratory/mixcr#1848] [Q&A]
What is the best preset or most suitable for analyzing TCR data from spatial sequencing generated with 10x Visium 3' chemistry, which has been subsequently amplified with specific primers following the Hudson protocol (PMID: 35707680)?

In version 3.0.13, they used the following command:

mixcr analyze shotgun -s hsa --starting-material rna --align "-OsaveOriginalReads=true" --assemble "--write-alignments" path/to/read2.fastq.gz mixcr_output/clones

What would be the equivalent analysis command using version 4.7 of MiXCR?

**Comment by mizraelson** [comment]
Hi,
The command should be:

```
mixcr analyze generic-ht-single-cell-amplicon-with-umi \
​     --rna \
     --species hsa \
     --tag-pattern "^(CELL:N{16})(UMI:N{12})(R1:*)\^(R2:*)"
     --floating-left-alignment-boundary \
     --floating-right-alignment-boundary C \
     --assemble-clonotypes-by CDR3 \
     -Massemble.consensusAssemblerParameters.maxConsensuses=10 \
     -Massemble.consensusAssemblerParameters.maxConsensuses=10 \
     sample_R1.fastq.gz \
     sample_R2.fastq.gz \
     sample_result
```

We are essentially using the generic single-cell command, allowing multiple clones per CELL barcode, as this is a spatial barcode. That being said, I remember analyzing the dataset from this paper, and there was significant noise due to V gene primers misannealing. Let me know how the command works on your end.
